.gitignore

```diff
@@ -0,0 +1,2 @@
+package-lock.json
+books
```

Comment:
You always should have a `.gitignore` file. Especially in a project like this where your script creates data files, you don't want git tracking changes in data files and clogging up the git history.